proto/redpanda/api/console/v1alpha1/debug_bundle.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
+
+package redpanda.api.console.v1alpha1;
+
+message CreateDebugBundleRequest {
+  int32 controller_logs_size_limit_bytes = 1;
+  int32 cpu_profiler_wait_seconds = 2;
+  string logs_since = 3; // TODO add validation 'YYYY-MM-DD', 'yesterday', or 'today'
+  int32 logs_size_limit_bytes = 4;
+  string logs_until = 5; // TODO add validation 'YYYY-MM-DD', 'yesterday', or 'today'
+  int32 metrics_interval_seconds = 6;
+  repeated string partition = 7;
```

Comment:
```suggestion
  repeated int32 partition = 7;
```

From the rpk docs:

```
Comma-separated partition IDs; when provided, rpk saves extra admin API requests for those partitions. Check help for extended usage
```

I think in our API we would want to expose it as []int32. Unsure if we want to do that as well in the admin api @michael-redpanda or remain closer to upstream rpk parameters there.